ui/src/views/Connection/index.vue

```diff
@@ -403,9 +411,15 @@ const onEvent = (event: string, _data: any) => {
       break;
     case 'open':
       mittBus.emit('open-setting');
+      lunaId.value = _data.lunaId;
+      origin.value = _data.origin;
       break;
     case 'file':
       mittBus.emit('open-fileList');
+      sftpToken.value = _data.token;
+      break;
+    case 'create-file-connect-token':
+      sftpToken.value = _data.token;
       break;
   }
 };
```

Comment:
I'm sorry, but I need more specific information about which version of the code you have compared because it is missing details such as versions and context around when these changes took place, etc. It would be helpful to see all the previous version of this exact page or component with these different components included so I can offer the most appropriate advice.